README.md

```diff
@@ -1,37 +1,47 @@
 # kmt-utilities
-Common middleware for Knowledge Manager Tools (KMT) microservices
+Common middleware for KAT microservices.
 
-## Installation guide
+KAT (Knowledge & administration tools) is a part of ft.com application created for Financial Times B2B clients.
 
-Include ``"kmt-utilities": "financial-times/kmt-utilities"``as a dependency in your package.json
+## Getting started
+To get a copy of the project up and running on your local machine for development and testing purposes run `$ git clone git@github.com:Financial-Times/kmt-utilities.git`.
```

Comment:
Done. Is in all other repos, I guess I just have forgotten to do in this one =)
Re make - will add when make functionality will be introduced (right now is still not in place), or the person who will merge make branch in.